logservice.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/urfave/cli"
+)
+
+var VERSION string
+
+var open = os.Open
+
+func scanFile(output io.Writer, file, buildID string) error {
+	body, err := open(file)
+	if err != nil {
+		return fmt.Errorf("Unexpected error opening the file: %v", err)
+	}
+
+	scanner := bufio.NewScanner(body)
+
+	for scanner.Scan() {
+		fmt.Fprintln(output, scanner.Text())
+	}
+	if err := scanner.Err(); err != nil {
+		return fmt.Errorf("Unexpected scan error: %v", err)
```

Comment:
This isn't a very useful addition to the error message that the scanner will give you. Either just return the error or add some context. The same for the error above.